src/components/PageWithSideMenu.tsx

```diff
@@ -65,6 +65,24 @@ export const NavigationList = ({ sx, ...props }: BoxProps) => (
   />
 );
 
+interface SelectableButtonProps extends ButtonProps {
+  isSelected?: boolean;
+}
+
```

Comment:
ikke nødvendigvis bare for `PageWithSideMenu`, så burde kanskje flyttes til egen fil?

Reply:
Flytter til egen fil, og lager egen PR for å oppdatere all bruk av denne, LinkButton, osv